### DISCUSSION TITLE
Netlify

### DISCUSSION THREAD

**Discussion by samosaman73** [thedev-id/thedev.id#91] [Q&A] [answered]
Does this support Netlify?

At the top it says it does not support Netlify, but towards the bottom it says about adding a custom domain into Netlify.

If Netlify is supported, how would I go about setting it up with a thedev.id subdomain?

**Comment by Codeize** [comment, marked as answer]
Hey! Sorry for the conflicting views, we DO NOT support Netlify as of now. Thanks for informing me of this issue.

**Reply by samosaman73** [reply]
In the cname.list file, I'm sure that I saw one of the subdomains pointing to Netlify though.

**Reply by Codeize** [reply]
Yep you did, Frans.thedev.id. this is the Co owners site.

**Comment by fransallen** [comment]
I'm currently working on this, also with services like Vercel and others. I'll let you know!
